components/tools/OmeroWeb/omeroweb/webgateway/marshal.py

```diff
@@ -134,9 +134,18 @@ def imageMarshal(image, key=None):
 
     # big images
     tiles = image._re.requiresPixelsPyramid()
-    width, height = image._re.getTileSize()
-    levels = image._re.getResolutionLevels()
-    zoomLevelScaling = image.getZoomLevelScaling()
+    rv['tiles'] = tiles
+    if (tiles):
```

Comment:
My request for tests is not specifically because of moving lines out of the `try` block. There are currently no tests for `imageMarshal()` at all. If we're going to change a marshalling method we should have tests for it. This is especially relevant when we know that `imageMarshal()` backs possibly the most significant and public JSON API endpoint we currently have (`.../imgData/`).

If you want to perform that testing by building an integration test against `.../imgData/` that seems fine.